### ISSUE TITLE
Opportunities created from project pages don't keep tags

### ISSUE THREAD

**Issue by csarsam** [18F/midas#780]
**Problem:** When a new opportunity is created from the `Add Opportunity` button on a Working Group page, the tags the user enters during the task creation wizard (topics, skills, time needed, people needed, location) do not appear on the task once it is created. This behavior does NOT happen when app from the `Browse Opportunities` page.

I experience this behavior locally (I am on commit 84be443), and on https://midas-dev.18f.us.

Steps to Reproduce:

* Navigate to a project page, and select `Add Opportunity`
* Complete each page of the task creation wizard. Be sure to include some tag data, like a topic or skill.
* Submit the new task
* View the new task. It will not have any of the tag data you added

---

I don't see any exceptions or bad request codes. This does not seem related to any existing issues.

**Comment by brendanwhitfield** [comment]
I believe that this is a client side problem. Here are the requests sent by the browser in both cases:

A successful task added using the `Browse Opportunities`:
```json
{
    "title": "qwer",
    "description": "qwerqwerqwer",
    "projectId": null,
    "tags": [{
        "name": "qwer",
        "type": "topic"
    }, {
        "name": "rewq",
        "type": "topic"
    }, {
        "name": "qwer",
        "type": "skill"
    }, {
        "name": "rewq",
        "type": "skill"
    }, 8, 4, 1, 15, 10],
    "name": null
}
```

and this is a bad request sent from the `Working Group` page:
```json
{
    "title": "asdf",
    "description": "asdfasdfasdf",
    "projectId": 2,
    "name": null
}
```

**Comment by brendanwhitfield** [comment]
The problem is in the difference between these files:

[assets/js/backbone/apps/tasks/list/controllers/task_list_controller.js](https://github.com/18F/midas/blob/devel/assets/js/backbone/apps/tasks/list/controllers/task_list_controller.js#L97-L100)
[assets/js/backbone/apps/browse/controllers/browse_list_controller.js](https://github.com/18F/midas/blob/devel/assets/js/backbone/apps/browse/controllers/browse_list_controller.js#L138-L142)

As you can see, `task_list_controller.js` is missing the tags field, as well as the `getTags()` helper. Rather than just copy/pasting, should these fields be moved to the `TaskModel` so this doesn't happen again?

**Comment by dhcole** [comment]
Closed by #840
